src/nanoFramework.Graphics/TouchPanel/Core/TouchPanel.cpp

```
@@ -133,7 +133,7 @@ HRESULT TouchPanelDriver::GetDeviceCaps(unsigned int iIndex, void *lpOutput)
 {
     if (lpOutput == NULL)
```

Clang (14.0.6)

[warning] use nullptr

(modernize-use-nullptr)